src/web/mjs/connectors/TopToon.mjs

```diff
@@ -15,39 +15,40 @@ export default class TopToon extends Connector {
     }
     async _getMangaFromURI(uri) {
         const request = new Request(uri, this.requestOptions);
-        const data = await this.fetchDOM(request, 'div.ver02 .tit_toon');
-        const id = uri.pathname + uri.search;
-        const title = data[0].textContent.trim();
-        return new Manga(this, id, title);
+        const data = await this.fetchDOM(request, 'div.bnr_episode_info p.tit_toon');
+        return new Manga(this, uri.pathname, data[0].textContent.trim());
     }
 
     async _getMangas() {
         const req = new Request('https://toptoon.com/hashtag', this.requestOptions);
-        const api = await this.fetchRegex(req, /jsonFileUrl\s*=\s*'([^']*)';/g);
+        const api = await this.fetchRegex(req, /fileUrl\s*:\s*'([^']+)'/g);
         const request = new Request(api[0], this.requestOptions);
         const data = await this.fetchJSON(request);
-        return data.map(ele => {
+        return data.map(item => {
             return{
-                id: ele.meta.comicsListUrl,
-                title: ele.meta.title
+                id: item.meta.comicsListUrl,
+                title: item.meta.title
             };
         });
     }
 
     async _getChapters(manga) {
         const request = new Request(new URL(manga.id, this.url), this.requestOptions);
-        const data = await this.fetchDOM(request, 'tr.episode_tr');
-        return data.map(ele => {
+        const data = await this.fetchDOM(request, 'div.episode_list ul a.episode-items');
+        return data.map(element => {
```

Comment:
Stay consistent, use "item" as line 27 (or the other way) :)

Reply:
Naming was by intention:
`element` => representing an HTML element from an HTML collection
`item` or `entry` => representing an object from a (json) array